src/faq_troubleshooting.md

```diff
@@ -0,0 +1,41 @@
+# Frequently Asked Questions and Troubleshooting
+
+**Q1: NCCL_ERROR_INVALID_ARGUMENT.**
+
+A1: Usually this is caused by the NCCL version conflict，update your torch==1.7.0 and torchvision==0.8.0.
```

Comment:
leave a space after comma

Reply:
Code should be in code format, like `torch>=1.7.0`.

Reply:
Don't use Q1, Q2, Q3,.. A1, A2, A3. Just make sections like in https://horovod.readthedocs.io/en/stable/troubleshooting_include.html